_posts/2023-08-30-introducing-mlstacks:-a-refreshed-way-to-deploy-mlops-infrastructure.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+author: "Alex Strick van Linschoten"
+title: "Introducing mlstacks: a refreshed way to deploy MLOps infrastructure"
+description: "Introducing mlstacks: a refreshed way to deploy MLOps infrastructure"
+category: zenml
+tags: mlstacks production mlops zenml evergreen tooling
+publish_date: August 30, 2023
+date: 2023-08-30T08:05:51Z
+thumbnail: /assets/posts/mlstacks/mlstacks_logo.png
+image:
+  path: /assets/posts/mlstacks/mlstacks_logo.png
+---
+**Last updated:** August 30, 2023
+
+We're excited to launch [the new `mlstacks` Python package](https://mlstacks.zenml.io). MLStacks builds on
+the work done to create Stack Recipes, allowing you to quickly spin up MLOps
+infrastructure using Terraform. It is designed to be used with ZenML, but can be
+used with any MLOps tool or platform.
+
+You can use `mlstacks` [directly as a Python package](https://pypi.org/project/mlstacks/), deploying infrastructure
+with the CLI after defining your stacks in YAML. Alternatively, you can let
+ZenML handle writing the stack and component specification files by [using the
+ZenML CLI](https://docs.zenml.io/stacks-and-components/stack-deployment) to define which components you want to deploy. We currently support
+modular MLOps stacks on AWS, GCP and K3D (for local use).
+
+We reworked the previous way of doing things to be more stable and reliable. We
+even added new features like the ability to get a cost estimate for your MLOps
+stacks. MLStacks is also designed to be able to work as a standalone deployment
+option independent of ZenML. All of this is available in the new `mlstacks`
+Python package and with ZenML's latest release (0.44.0).
+
```

Comment:
Maybe add similar works like matcha and say we were inspired but wanted to do it differently?